tests/dynamic-policy.spec.js

```diff
@@ -0,0 +1,78 @@
+const {
+    expect
+} = require("chai");
+const fs = require("fs");
+const nock = require("nock");
+const uuid = require("uuid");
+
+const config = require("../config");
+const {
+    DynamicPolicyRequest,
+    Policy,
+    getDynamicPolicy,
+} = require("../src/dynamic_policy_service");
+const Payload = require('../src/request/payload').Payload;
+
+const privateKeyFile = fs.readFileSync('./tests/sample-data/keys/node-sdk-test.pem', 'utf8');
+
+const APP_ID = "9cd51c3f-f03a-46e3-aa9b-244cbfc99d86";
```

Comment:
We don't want to check in App IDs and .pem files to the public repos, even if it's only in a test. Do you want your tests to be calling the live service? Will a mock service not suffice?